src/handlers/notebook.ts

```diff
@@ -15,13 +15,30 @@ import { Debugger } from '../debugger';
 
 import { EditorHandler } from './editor';
 
+import { ToolbarButton } from '@jupyterlab/apputils';
 import { IDebugger } from '../tokens';
 
 export class NotebookHandler implements IDisposable {
   constructor(options: NotebookHandler.IOptions) {
     this.debuggerModel = options.debuggerService.model as Debugger.Model;
     this.debuggerService = options.debuggerService;
     this.notebookPanel = options.widget;
+    this.notebookPanel.toolbar.insertAfter(
```

Comment:
The code to add a new button is almost the same as for the console (and later for the editors too).

There is an issue to create a function for adding debug buttons in https://github.com/jupyterlab/debugger/issues/154. Should we refactor that in this PR or in a separate change?

Reply:
Good catch. I will try to refactor this PR.